### PR TITLE
Update BO theme style in legacy pages to match the UI kit

### LINKED ISSUE #22435: [Legacy default theme] - Improve style consistency of legacy pages
#### Is your feature request related to a problem?

Legacy pages have old styles, while new theme received recent update (UIKit), which could be a bit improved in order to win some consistency, even if they are temporary

#### Describe the solution you'd like

We would require to change some styles :

- Buttons
- Forms
- Alerts
- Inputs
- Cards

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Default theme style needs to match with new-theme style, so it's not a shock when you go through these different contexts
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #22435.
| How to test?      | Test every legacy pages, the style should be almost the same as migrated pages (almost, we can't do exactly the same)
| Possible impacts? | Every legacy pages

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23146)

### CODE REVIEW

File: admin-dev/themes/default/scss/partials/_buttons.scss [hunk 1]
```diff
@@ -1,15 +1,81 @@
...
+    &,
+    i {
+      color: $medium-gray;
+      transition: 0.25s ease-out;
+    }
+
     &:hover {
-      background-color: $primary-light-color;
-      border-color: $primary-light-color;
+      color: #fff;
```
Comment (PierreRambaud) [inline admin-dev/themes/default/scss/partials/_buttons.scss]: Why not using a new variable like $btn-color?

**Review by PierreRambaud — DISMISSED** [review]
I approve even if I've one question. Can improve later if needed

**Review by Progi1984 — DISMISSED** [review]

File: admin-dev/themes/default/scss/partials/_buttons.scss (hunk 1 above)
Comment (NeOMakinG) [reply admin-dev/themes/default/scss/partials/_buttons.scss]: It could be a thing yes

**Comment by sarahdib** [comment]
@NeOMakinG The button are a little oversised :) 
Capture d’écran 2021-02-10 à 21 49 18
Capture d’écran 2021-02-10 à 21 49 55
Capture d’écran 2021-02-10 à 21 50 15
Capture d’écran 2021-02-10 à 21 50 59
Capture d’écran 2021-02-10 à 21 51 46
Capture d’écran 2021-02-10 à 21 52 06
Capture d’écran 2021-02-10 à 21 53 19
Capture d’écran 2021-02-10 à 21 55 27
Capture d’écran 2021-02-10 à 21 55 52

The button are a little oversised and saved button is not at the right place
Capture d’écran 2021-02-10 à 21 58 37

**Review by PierreRambaud — DISMISSED** [review]

File: admin-dev/themes/default/scss/partials/_buttons.scss (hunk 1 above)
Comment (Progi1984) [reply admin-dev/themes/default/scss/partials/_buttons.scss]: Ping @NeOMakinG

**Review by Progi1984 — COMMENTED** [review]
A last one

**Comment by NeOMakinG** [comment]
@Progi1984 I changed buttons colors into vars

**Review by PierreRambaud — DISMISSED** [review]

**Comment by Progi1984** [comment]
@NeOMakinG Needs a rebase

**Review by PierreRambaud — DISMISSED** [review]

**Review by matthieu-rolland — DISMISSED** [review]

**Review by matks — DISMISSED** [review]

**Comment by sarahdib** [comment]
@NeOMakinG 
With this pr : 
Capture d’écran 2021-02-22 à 08 36 12
without :
Capture d’écran 2021-02-22 à 08 43 48

**Comment by NeOMakinG** [comment]
@sarahdib should be fine now

**Review by Progi1984 — DISMISSED** [review]

**Review by atomiix — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]

**Comment by sarahdib** [comment]
@NeOMakinG 

Capture d’écran 2021-02-26 à 08 41 40
Capture d’écran 2021-02-26 à 08 57 26

**Comment by NeOMakinG** [comment]
Fixed it and another bug on translatable dropdown heights

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @NeOMakinG & @sarahdib

**Pull request merged**